### PR TITLE
Added file input types.

### PR DESCRIPTION
**Pull request by raquelarrojo**

### CODE REVIEW

File: lib/src/file-input/types.ts [hunk 1]
```diff
@@ -8,56 +7,56 @@ type Margin = {
...
   /**
-   * Defines the file types accepted by the component. It is not possible to select a file with a non valid type.
+   * The file types that the component accepts. Its value must be one of all the possible values of the HTML file input's accept attribute.
    */
   accept?: string;
   /**
-   * An array of FileData representing the selected files. 
+   * An array of files representing the selected files.
    */
   value?: FileData[];
```
Comment (GomezIvann) [inline lib/src/file-input/types.ts:42]: Does an uncontrolled file input make sense? Maybe `callbackFile` or/and `value` should be mandatory, what do you think?

**Review by GomezIvann — CHANGES_REQUESTED** [review]

File: lib/src/file-input/types.ts [hunk 2]
```diff
@@ -8,56 +7,56 @@ type Margin = {
 };
 type FileData = {
```
[the remark this answers is not in the document]
Comment (raquelarrojo) [reply lib/src/file-input/types.ts:8]: Agree!

File: docs/src/pages/components/cdk-components/file-input/api.jsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ const fileInputPropsTable = () => {
           <th>Description</th>
         </tr>
         <tr>
-          <td>name</td>
+          <td>name: string</td>
           <td></td>
           <td>Name attribute.</td>
         </tr>
```
[the remark this answers is not in the document]
Comment (raquelarrojo) [reply docs/src/pages/components/cdk-components/file-input/api.jsx:17]: Do yo refer to mode type in types.ts right? It already has its type set to `"file" | "filedrop" | "dropzone"`. Also in the documentation it's set like that.

File: lib/src/file-input/types.ts (hunk 1 above)
Comment (raquelarrojo) [reply lib/src/file-input/types.ts:42]: Yes, file input is always controlled. I will change both to required!

File: docs/src/pages/components/cdk-components/file-input/api.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (GomezIvann) [reply docs/src/pages/components/cdk-components/file-input/api.jsx:17]: I referred to the docs. Currently, the type is like this: 

image

In the other components, these kinds of types are written as above.

[the remark this answers is not in the document]
Comment (raquelarrojo) [reply docs/src/pages/components/cdk-components/file-input/api.jsx:17]: Got it! Done!

**Review by GomezIvann — APPROVED** [review]

**Pull request merged**